### PR TITLE
telemetry: log when sending telemetry

### PR DESCRIPTION
**Pull request by dveeden**
### What problem does this PR solve?

Problem Summary:
Add logging about sending telemetry data to make it more obvious.

### What is changed and how it works?

What's Changed:

- Adding logging when sending telemetry data
- Adding logging about telemetry configuration on startup
- Send telemetry on startup

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Inspect logging

### Release note 

- No release note

### CODE REVIEW

File: telemetry/telemetry.go [hunk 1]
```diff
@@ -127,6 +129,7 @@ func reportUsageData(ctx sessionctx.Context, etcdClient *clientv3.Client) (bool,
 	}
 
 	req.Header.Add("Content-Type", "application/json")
+	logutil.BgLogger().Info(fmt.Sprintf("Uploading telemetry data to %s", apiEndpoint))
```
Comment (lzmhhh123) [inline telemetry/telemetry.go:132]: How about also adding the report interval in the log?

Comment (dveeden) [reply telemetry/telemetry.go:132]: Maybe that should be logged at startup? e.g.:
```telemetry=enabled apiEndpoint="https://telemetry.pingcap.com", interval=6h```? 

And then keep this line as is when actually sending the report?

Comment (lzmhhh123) [reply telemetry/telemetry.go:132]: Yep. This way LGTM.

Comment (dveeden) [reply telemetry/telemetry.go:133]: @lzmhhh123 I've added the reporting of the config on startup. This also does an initial run on startup now.

**Comment by lzmhhh123** [comment]
@dveeden Could you fix the lint? The rest LGTM.

**Review by lzmhhh123 — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**